python/tests/test_feature_view.py

```diff
@@ -176,3 +176,21 @@ def test(col1):
         transformation_functions = fv.transformation_functions
 
         assert transformation_functions[0] != transformation_functions[1]
+
+    def test_logging_enabled_instances(self, mocker, backend_fixtures):
```

Comment:
could add some tests for the new methods in FV they look complex enough to break easily :D 
ex:
```
    @property
    def _untransformed_feature_names(self) -> List[str]:
        """Get untransformed feature names."""
        if self.__untransformed_feature_names is None:
            self.__untransformed_feature_names = [
                feature.name
                for feature in self.features
                if feature.name not in self._label_column_names
                and feature.name not in self.training_helper_columns
                and feature.name not in self.inference_helper_columns
            ]
        return self.__untransformed_feature_names
```